src/test/scala/org/neo4j/spark/Neo4jSparkTSE.scala

```diff
@@ -85,18 +74,18 @@ class Neo4jSparkTest {
   }
 
   @Test def runCypherQueryWithPartition() {
-    val neo4j: Neo4j = Neo4j(sc).cypher("MATCH (n:Person) RETURN id(n) SKIP {_skip} LIMIT {_limit}").partitions(4).batch(25)
+    val neo4j: Neo4j = Neo4j(sc).cypher("MATCH (n:Person) RETURN id(n) SKIP $_skip LIMIT $_limit").partitions(4).batch(25)
```

Comment:
did we also change the test inside of the execution/partition?
and the docs in the readme?

Reply:
yes in both of them